GSoC2019/src/index.html

```diff
@@ -10,15 +11,38 @@
   <script src="http://unpkg.com/d3-force"></script>
 
   <link rel="stylesheet" href="css/forced3.css" type="text/css">
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  
   <script src="js/jquery-3.2.1.min.js"></script>
   <script src="js/highcharts.js"></script>
   <script src="js/exporting.js"></script>
   <script src="js/export-data.js"></script>
-   <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
-<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
-<link rel="stylesheet" href="css/materialize.css" type="text/css">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="css/materialize.css" type="text/css">
   <!--<script src="../../dist/force-graph.js"></script>-->
+  
+  <!--Jquery-->
+  <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css">  
+  <script src="https://code.jquery.com/jquery-3.4.1.js"></script>  
```

Comment:
Is it necessary to pull in this version of `jquery`?  Note that there's already one stored in the `js` directory.

Reply:
Yeah. Its not necessary to pull this version for now.